File: go/uri1008/uri1008.go
package main

import "fmt"

func main() {

	var numero int
	var horasTrabalhadas int
	var valorHora float64

	fmt.Scanf("%d", &numero)
	fmt.Scanf("%d", &horasTrabalhadas)
	fmt.Scanf("%f", &valorHora)

	salario := float64(horasTrabalhadas) * valorHora

	fmt.Printf("NUMBER = %d\n", numero)
	fmt.Printf("SALARY = U$ %.2f\n", salario)
}

File: c#/Uri1008.cs
using System; 

class Uri1008 
{
    static void Main(string[] args) 
    { 
        int numero = int.Parse(Console.ReadLine());
        int horas = int.Parse(Console.ReadLine());
        double valor = double.Parse(Console.ReadLine());

        double salario = valor * horas;

        Console.WriteLine($"NUMBER = {numero}");
        Console.WriteLine($"SALARY = U$ {salario:F2}");
    }
}